Changes.md

```diff
@@ -1,3 +1,10 @@
+1.1.x.x (relative to 1.1.9.0)
```

Comment:
The `1.1.x` series is receiving bugfixes only now, so this PR needs to be rebased and retargeted to `1.2_maintenance`. The rebase should also get rid of the unwanted merge commits in this PR.